app/routes/pages/components/Sidebar.css

```diff
@@ -24,9 +23,13 @@
 }
 
 .sidebarHeader {
-  font-size: 2em;
-  margin-bottom: -10px;
+  font-size: 1.8em;
+  margin: 20px 0 0px 0;
   font-weight: 600;
+  width: 95%;
+  text-align: center;
+  border-bottom: 2px solid var(--additive-background);
```

Comment:
I feel like the border color should be `var(--border-gray)` so that it matches the rest of the webapp, but could you maybe instead tweak it to be lighter and better looking? I never changed it up.